RELEASE/scripts/autoscend/iotms/mr2023.ash

```diff
@@ -432,7 +432,7 @@ void auto_scepterSkills()
 		{
 			if(!is100FamRun())
 			{
-				use_familiar($familiar[Mosquito]); //equipping Mosquito so we don't get a big rock
+				use_familiar(findNonRockFamiliarInTerrarium()); //equipping any non-rock fam so we don't get a big rock
```

Comment:
rocks actually get the consolation ribbon. Casting it with no familiar gets the big rock.